.Gruntfile.babel.js

```diff
@@ -337,7 +337,7 @@ module.exports = (grunt) => {
           match: /\.scss$/,
           recurse: true
         }),
-        eslint({ throwOnError: true, throwOnWarning: true }),
+        eslint({ throwOnError: true, throwOnWarning: true, fix: true }),
```

Comment:
NOTE: 👮‍♀ I won't forget to delete this before merging

Reply:
seems about time to remove :)